src/firefly/js/ui/FireflyRoot.jsx

```diff
@@ -1,31 +1,68 @@
-import React from 'react';
-import {CssBaseline, ScopedCssBaseline, CssVarsProvider, useColorScheme, GlobalStyles} from '@mui/joy';
-import {getTheme} from './ThemeSetup.js';
-
-
+import React, {useEffect} from 'react';
+import {ScopedCssBaseline, extendTheme, CssVarsProvider, useColorScheme, GlobalStyles} from '@mui/joy';
 
+import {defaultTheme} from './ThemeSetup.js';
+import {getAppOptions} from '../core/AppDataCntlr.js';
+import {logger} from '../util/Logger.js';
 
+import '@fontsource/inter/200.css'; // Lighter
```

Comment:
So we don't have to have this in package.json? it just pulls it in?

Reply:
After looking into this, I realized that you already included the npm package `@fontsource/inter`.  All you need is to import the different weight css files to use it.

Reply:
So are you going to change package.json? or is it fine?